pom.xml

```diff
@@ -68,6 +68,30 @@
                 <artifactId>gson</artifactId>
                 <version>2.10.1</version>
             </dependency>
+            <!--
+                Upgrade log4j version to fix security vulnerability
+                ref - https://logging.apache.org/log4j/2.x/security.html
+            -->
+            <dependency>
+                <groupId>org.apache.logging.log4j</groupId>
+                <artifactId>log4j-api</artifactId>
+                <version>2.17.1</version>
```

Comment:
Can define.a <log4j.version> under <properties> and use it.

Reply:
noted